### DISCUSSION TITLE
Fly io Litefs Cloud with Pocketbase

### DISCUSSION THREAD

**Discussion by milimyname** [pocketbase/pocketbase#2857] [Q&A] [answered]
I wonder if anybody used Fly io Litefs Cloud with Pocketbase?

Why am I asking it?
I deployed my Pocketbase app in the ams region but I want to optimize users e.g closer to North America or Asia that’s why I’ll clone it to those regions.

As it is now, a pocketbase app needs a persistant volume which will be cloned to other regions but they won’t be replicated.

I know that Pocketbase ain’t for HA but this article might help me solve the problem but I haven't tried it yet.
https://fly.io/docs/litefs/speedrun/

Btw, I asked this question in the fly io community but I think it might make sense to ask here as well.

**Comment by ganigeorgiev** [comment, marked as answer]
Keep in mind that while you can use [litefs](https://github.com/superfly/litefs) or [marmot](https://github.com/maxpert/marmot) to replicate the database, PocketBase wasn't designed with multiple servers in mind and some things may not work as expected even if you sync the db files across the servers - custom event hooks, realtime subscriptions and anything else that relies on the `core.App` runtime state since technically you'll have multiple separate applications and there is no builtin mechanism at the moment that will keep them in sync.

**Comment by milimyname** [comment]
Thank you, Mr @ganigeorgiev. I'll give a shot later but update it as well how it went.
